.github/workflows/pull-request.yml

```diff
@@ -44,10 +44,28 @@ jobs:
       
       - name: Build
         run: dotnet build Letterbook.sln --configuration Release
+
+      - name: Install Playwright Browsers
+        run: pwsh Letterbook.Web.Tests/bin/Release/net8.0/playwright.ps1 install
+
+      - name: Docker compose up
+        run: docker compose up -d
```

Comment:
If we use github actions built-in service containers, we won't have to manage a docker compose lifetime ourselves.
https://docs.github.com/en/actions/using-containerized-services/about-service-containers

Reply:
Done using this one-liner, hope that's okay

hoverkraft-tech/compose-action@v2.0.1